client/src/app/containers/Login/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Link from '@material-ui/core/Link';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { useHistory } from 'react-router-dom';
+
+import { useAuth } from '../../../auth/useAuth';
+
+const useStyles = makeStyles(theme => ({
+    title: {
+        fontSize: '2rem',
+    },
+    paper: {
+        marginTop: theme.spacing(8),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
```

Comment:
i will never support ie with this project

Reply:
I think this is more copypasta; unclear if this is needed in all browsers or not. Will investigate.